t04_refactored.py

```diff
@@ -139,8 +139,46 @@ def team_3_adv():
 
 
 def team_4_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1jnB2QAcNxvVUlpapvX8l4pWjCuHqAA9jRbz7FBr6J0M/edit?usp=sharing
+    Aije Ebosele
+    Sara Bako
+    :return:
+    """
+
+    delay = 1
+
+
+    username = input("Please enter username")
+    print("Welcome,", username, ", to Generic Fantasy College™.")
+    sleep(delay)
+    print("You are awoken by your alarm")
+    print("It is 8:00AM")
+    print()
+    sleep(delay * 2)
+    print("Looking through your window the sky has a magenta hue")
+    print("Staying here is certainly not wise. You must find your way out.")
+    print("\n")
+    sleep(delay)
+
+    sleepChoice = input("Do you wake up, be late, or sleep?")
+
+    if sleepChoice == "wake up":
+        # Good Choice
+        print("You go to classes and are a productive member of society")
+        sleep(delay)
```

Comment:
Any branch of the if/elif/else statement that you want your user to live in should end with `return True`. `return False` to kill your user.